contracts/Core/StakeManager.sol

```diff
@@ -312,9 +312,9 @@ contract StakeManager is Initializable, StakeStorage, StateManager, Pause, Stake
         // https://soliditydeveloper.com/stacktoodeep
         {
             (uint16 bountyNum, uint16 burnSlashNum, uint16 keepSlashNum) = (slashNums.bounty, slashNums.burn, slashNums.keep);
-            bounty = (_stake * bountyNum) / baseDenominator;
-            amountToBeBurned = (_stake * burnSlashNum) / baseDenominator;
-            amountToBeKept = (_stake * keepSlashNum) / baseDenominator;
+            bounty = (_stake * bountyNum) / BASE_DENOMINATOR;
+            amountToBeBurned = (_stake * burnSlashNum) / BASE_DENOMINATOR;
+            amountToBeKept = (_stake * keepSlashNum) / BASE_DENOMINATOR;
```

Comment:
Does it make sense to optimise this by storing `_stake / BASE_DENOMINATOR`?

Reply:
yeah, you mean to store _stake/BASE_DENOMINATOR  as variable and multiply with corresponding numerator?

variable = _stake/BASE_DENOMINATOR
bounty = variable * bountyNum ...

right? @SkandaBhat

Reply:
Yep